Bottom: 145fb576855a3faea61615258ae09ce8afc1cf8e
Top:    68c8b4b1284b8f1c74cb3c79d8877c02a40c0782
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:59 -0500

Create certificate directories during image build


---
diff --git a/docker/docker-svn/Dockerfile b/docker/docker-svn/Dockerfile
index 967bca5c..7397d68b 100644
--- a/docker/docker-svn/Dockerfile
+++ b/docker/docker-svn/Dockerfile
@@ -10,6 +10,7 @@ MAINTAINER Joseph Lutz <Joseph.Lutz@novatechweb.com>
 RUN apt-get update && \
   DEBIAN_FRONTEND=noninteractive apt-get install --yes --no-install-recommends \
     apache2 \
+    ca-certificates \
     enscript \
     libapache2-svn \
     php5-common \
diff --git a/docker/docker-svn/configure.sh b/docker/docker-svn/configure.sh
index fab90adb..21c3114a 100755
--- a/docker/docker-svn/configure.sh
+++ b/docker/docker-svn/configure.sh
@@ -6,6 +6,11 @@ SVN_BASE_DIR=/var/lib/svn
 # Create a directory for a dead symlink
 mkdir /var/cache/websvn/tmp
 
+# Create certificate directories
+mkdir -p /etc/grid-security/certificates
+mkdir -p /etc/ssl/certs/
+mkdir -p /etc/ssl/private
+
 # comment out apache2 config file lines that reference the environment variables
 sed -i 's|^Mutex file|#Mutex file|' /etc/apache2/apache2.conf
 sed -i 's|^PidFile |#PidFile |' /etc/apache2/apache2.conf
